.travis.yml

```diff
@@ -10,12 +10,11 @@ before_install:
   - export CXX=g++-4.8
   - printenv
   - sudo cp .travis/config.hpp /usr/include/armadillo_bits/config.hpp
-install:
-  - mkdir build && cd build && cmake -DDEBUG=OFF -DPROFILE=OFF .. && make -j4
-script:
-  - travis_wait 30 ./bin/mlpack_test -p
-notifications:
-  email:
-    - mlpack-git@cc.gatech.edu
-  irc:
-    - "chat.freenode.net#mlpack"
+  - pip install --user cpp-coveralls
+  - sudo apt-get install lcov
+  - sudo apt-get install ruby
+  - sudo gem install coveralls-lcov
+  - printf "Y\n" > apt_input.txt
```

Comment:
This line shouldn't be necessary anymore. :)  You can also use `apt-get -y` if you need, or even `yes | apt-get install ...`.